mitosheet/src/mito/types.tsx

```diff
@@ -964,6 +967,7 @@ export enum ActionEnum {
     Rename_Column = 'rename column',
     Rename_Dataframe = 'rename dataframe',
     Rename_Graph = 'rename graph',
+    Scatter_Plot = 'scatter graph',
```

Comment:
I would be down to rename these actions to GRAPH_BAR, GRAPH_LINE, etc -- just so alphabetically they are close.

Reply:
And moving in the actions.tsx file.